plugins/pulp_rpm/yum_plugin/util.py

```diff
@@ -184,8 +181,9 @@ def errata_format_to_datetime(datetime_str, msg):
     Convert known errata date-time formats to datetime object.
 
     Expected formats are:
-     - '%Y-%m-%d %H:%M:%S'
-     - '%Y-%m-%d %H:%M:%S UTC'
+    - '%Y-%m-%d %H:%M:%S UTC'
+    - '%Y-%m-%d %H:%M:%S'
+    - '%Y-%m-%d'
```

Comment:
Good additional format. Maybe QE could audit some of the repos from RHEL, Fedora, and EPEL to see if there are other format strings which would be useful to include. This code would allow them to do that based on filtering of the WARNING log statements.